src/components/Dashboard/NoRewards.js

```diff
@@ -1,12 +1,20 @@
 import React from 'react'
-import MessageCard from '../MessageCard'
+import { useWallet } from 'use-wallet'
 
+import MessageCard from '../MessageCard'
+import { useJurorDraftRewardedQuery } from '../../hooks/query-hooks'
 import iconNoRewardsSvg from '../../assets/IconNoRewards.svg'
 
 function NoRewards() {
+  const wallet = useWallet()
+  const hasJurorClaimedRewards = useJurorDraftRewardedQuery(
+    wallet?.account.toLowerCase()
```

Comment:
I don't think we handle the case where the `wallet.account` doesn't exist in `useJurorDraftRewardedQuery`. Should it?

As well, do subscriptions require accounts to be in lower case (because that's how they're encoded in the subgraph)? If so, that also sounds like something the query function should handle for us transparently.

Reply:
> I don't think we handle the case where the wallet.account doesn't exist in useJurorDraftRewardedQuery. Should it?

Well actually this component is only rendered in the case that an account is connected to the dashboard so the `wallet?.` check doesn't make sense in this case. Will update and add a comment.

> As well, do subscriptions require accounts to be in lower case (because that's how they're encoded in the subgraph)? If so, that also sounds like something the query function should handle for us transparently. 

Yes, that is the reason, will look if urql provides functions before sending the request.